api-module-library/slack/api.js

```diff
@@ -81,7 +81,8 @@ class Api extends OAuth2Requester {
             return this._request(url, options, i + 1);
         } else if (
             parsedResponse.error === 'invalid_auth' ||
-            parsedResponse.error === 'auth_expired'
+            parsedResponse.error === 'auth_expired' ||
+            parsedResponse.error === 'token_expired'
```

Comment:
not sure if auth_expired exists for slack?

Reply:
I think it was returned in a test at one point? Not in the docs anywhere though, agreed.